datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import logging
+import os
+import pathlib
+import typing
+
+import requests
+from google.cloud import storage
+
+
+def main(
+    base_url: str,
+    folder: pathlib.Path,
+    version: str,
+    gcs_bucket: str,
+    target_gcs_folder: str,
+    pipeline: str,
+):
```

Comment:
Always incorporate a return type.  For no return type use:

def fn(...) -> None:

Reply:
Ok done